source/stylesheets/base/_variables.scss

```diff
@@ -1,18 +1,29 @@
 // Breakpoints
-$breakpoint-medium: 480px;
-$breakpoint-large: 890px;
+$breakpoint-medium: 40em;
+$breakpoint-large: 55em;
 
 // Colors
-$teal: #12939A;
+$teal: #118288;
 $teal-dark: mix($teal, black, 80%);
+$teal-transparent: rgba(17, 130, 136, 0.7);
+$very-light-gray: #F2F2F2;
+$light-gray: #ccc;
+$white: #FFF;
 
 // Font Sizes
 $font-size-base: 1rem;
-$font-size-logo: 2rem;
+$font-size-smallest: $font-size-base * 0.6;
+$font-size-small: $font-size-base * 0.75;
+$font-size-medium: $font-size-base * 1.25;
+$font-size-large: $font-size-base * 1.5;
+$font-size-x-large: $font-size-base * 2;
+$font-size-xx-large: $font-size-base * 3;
```

Comment:
If the larger sizes are getting tshirt sizes, consider using that same convention for the smaller sizes, too, for consistency and scalability. For example:
`$font-size-x-small`

Reply:
I'd agree with making the naming convention more consistent.

Another direction you might consider is making the naming math based. If `1rem` is your base, then all names would flow from that `$font-size-two: 2rem;` $font-size-one-and-quarter: 1.25rem`, `$font-size-three-quarters`, etc.

What this fixes is that moment when you realize you need a size between, say `2rem` and `3rem`. Right now, you're stuck and either changing the value of `xx-large` or creating something like `x-larger` or deviating from the convention.